docs/quickstart.md

```diff
@@ -109,28 +109,62 @@ $ iasql install --db startup aws_cloudwatch aws_ecr aws_ecs aws_elb aws_security
 
 ## Connect to your db and provision cloud resources
 
-1. Install `psql` in your command line by following the instructions for your corresponding OS [here](https://www.postgresql.org/download/)
-
-2. Get a local copy of the [SQL script](https://github.com/iasql/quickstart/blob/main/quickstart.sql) hosted in the repository for this quickstart
+1. Get a local copy of the [quickstart repository](https://github.com/iasql/quickstart/blob/main/quickstart.sql)
 
 ```bash
 git clone git@github.com:iasql/quickstart.git
 cd quickstart
 ```
 
-3. Run the script on your db by invoking `psql` with the connection string provided on db creation and set the desired `project-name` parameter that your resources will be named after:
+2. Install the `typeorm` CLI and the Node.js project that uses TypeORM under the `quickstart/infra` folder
 
 ```bash
-psql postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4 -v project_name="'quickstart'" -f quickstart.sql
+cd infra
+npm i
+npm -g i typeorm
 ```
 
+3. (Optional) Set the desired project name that your resources will be named after by changing the `name` in the `quickstart/infra/package.json`. If the name is not changed, `quickstart` will be used.
+
 :::note
 
 The `project-name` can only contain alphanumeric characters and hyphens(-) because it will be used to name the load balancer
 
 :::
 
-4. Apply the changes described in the db to your cloud account
+3. Create a [`ormconfig.json`](https://typeorm.io/#/using-ormconfig/using-ormconfigjson) with the connection parameters provided on db creation. In this case:
```

Comment:
maybe we can provide this file with all fields commented?

Reply:
we display it below and Docusaurus gives you the copy prompt :)